src/client.js

```diff
@@ -94,6 +100,15 @@ function createMethod(clientMethod, dbg, cancelCache) {
 
         call.pipe(though2.obj(onData, onEnd));
         call.on('error', onError);
+        domain.once('error', onError);
+
+        const originalUnsub = observer.unsubscribe;
+        observer.unsubscribe = function(...args) {
+          grpcCancel();
```

Comment:
From my testing this line is not needed, I think since we call `call.destroy()` the cancelCache flow happens anyway.

Reply:
I was seeing dangling server connections without it.

Reply:
🤔